CHANGES.md

```diff
@@ -133,6 +133,8 @@ To be released.
      -  `BlockStates` (`0x23`)  [[#946]]
      -  `DifferentVersion` (`0x30`)  [[#949]]
  -  Every message now contains app protocol version in its header.  [[#949]]
+ -  Removed `PreloadBlockDownloadFailEventArgs` class.
+ -  Removed `blockDownloadFailed` parameter from `Swarm<T>.PreloadAsync()` method.
```

Comment:
We don't need an issue number here?
```suggestion
 -  Removed `PreloadBlockDownloadFailEventArgs` class. [[#1002]]
 -  Removed `blockDownloadFailed` parameter from `Swarm<T>.PreloadAsync()` method. [[#1002]]
```

Reply:
nvm. same at https://github.com/planetarium/libplanet/pull/1002#discussion_r487644554